topydo/lib/Filter.py

```diff
@@ -214,3 +214,35 @@ def match(self, p_todo):
 
         return False
 
+PRIORITY_MATCH = r"\((?P<operator><=?|>=?|=|!)?(?P<priority>[A-Z]{1})\)"
+
+class PriorityFilter(Filter):
+    def __init__(self, p_expression):
+        super(PriorityFilter, self).__init__()
+
+        self.expression = p_expression
+
+        match = re.match(PRIORITY_MATCH, self.expression)
+        if match:
+            self.operator = match.group('operator') or '='
+            self.priority = match.group('priority')
+
+    def match(self, p_todo):
+        operand1 = p_todo.priority()
+        operand2 = self.priority
+
+        if self.operator == '<':
```

Comment:
Nice, this is what I had in mind.

If you add the tests + docstrings I'll merge it in.